### PR TITLE
Validate VALK

### PR DESCRIPTION
**Pull request by SolanaValkyrie**
# Validate [VALK](https://solscan.io/token/84KY3exPaKV3b49YowaW2E71DrVjYVLog9vdFXZYfokA)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: [https://twitter.com/SolanaValkyrie/status/1721481245060915327](https://twitter.com/SolanaValkyrie/status/1721481245060915327)
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [ ] The metadata provided in the PR matches what is on-chain (Mandatory)
- [ ] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [ ] My change matches the format in the file (no spaces between fields).
- [ ] My token is already live and trading on Jupiter.
- [ ] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -796,4 +796,5 @@ Flowmatic,FM,Eh1fXbAipe4k7CYR9UMb2bbWmBcpU3HcyX3LWuRVFBLz,4,https://nftstorage.l
 SolBird,BIRD,7y2KpMzsxmjCN8ogA3PSfwZvmYUgMuWo7NWY4Ct5jRHp,6,https://arweave.net/K_IaCXwL9YQTPJnZfvi302k2lhANDvmzkiy6e0Y6Zho,true
 Soltopian,Gorilla,61AaYTDtQdXx5wEWNQep37HYKBEJaTkAMEs9c2AnFdx8,9,https://arweave.net/XhknuWjgQF57-9SFgQIRh9NlcWEQJvzVM2qd45O7hpU,true
 Spore,SPORE,6Cw88QKMxS7LzAucXbYDcUEXPq1TvsojYPV9xWwJPt7r,8,https://bafkreidodmcqrprsgoelphtr5vyqintuqscwixqfh6ndaqurvp2mnnx2h4.ipfs.nftstorage.link/,true
-Render Token,RENDER,rndrizKT3MK1iimdxRdWabcF7Zg7AR5T4nud4EkHBof,8,https://shdw-drive.genesysgo.net/5zseP54TGrcz9C8HdjZwJJsZ6f3VbP11p1abwKWGykZH/rndr.png,true
\ No newline at end of file
+Render Token,RENDER,rndrizKT3MK1iimdxRdWabcF7Zg7AR5T4nud4EkHBof,8,https://shdw-drive.genesysgo.net/5zseP54TGrcz9C8HdjZwJJsZ6f3VbP11p1abwKWGykZH/rndr.png,true
+Valkyrie,VALK,84KY3exPaKV3b49YowaW2E71DrVjYVLog9vdFXZYfokA,9,https://arweave.net/qXcUvMYTFqiJEWosnydID47TRyAhiYpii-Y0n_FEtts,true
```
Comment (Timoon21) [inline validated-tokens.csv:800]:
```suggestion
VALKYRIE,VALK,84KY3exPaKV3b49YowaW2E71DrVjYVLog9vdFXZYfokA,9,https://arweave.net/qXcUvMYTFqiJEWosnydID47TRyAhiYpii-Y0n_FEtts,true
```

**Comment by SolanaValkyrie** [comment]
Waiting for another review. please our community is really need this :'(

**Comment by SolanaValkyrie** [comment]
still waiting :(

**Comment by meijiao** [comment]
hi, i saw less than $5 in volume traded over the last 24 hours. Closing this PR, your token is still available for trading on the 'All' list as long as liquidity requirements have been met, and you can reopen this PR when there is more traction.

**Pull request closed without merging**